src/languages/en-US.ts

```diff
@@ -880,6 +880,7 @@ export default class extends Language {
 			NONE: 'None'
 		},
 		COMMAND_FFXIV_NO_CHARACTER_FOUND: 'I am sorry, but I was unable to find a character with that name.',
+		COMMAND_FFXIV_INVALID_SERVER: 'I am sorry, but an invalid server name was passed. Please remember that it is case-sensitive.',
 		COMMAND_FFXIV_NO_ITEM_FOUND: 'I am sorry, but I was unable to find an item with that query.',
```

Comment:
```suggestion
		COMMAND_FFXIV_NO_CHARACTER_FOUND: `${REDCROSS} I am sorry, but I was unable to find a character with that name.`,
		COMMAND_FFXIV_INVALID_SERVER: `${REDCROSS} I am sorry, but an invalid server name was passed. Please remember that it is case-sensitive.`,
		COMMAND_FFXIV_NO_ITEM_FOUND: `${REDCROSS} I am sorry, but I was unable to find an item with that query.`,
```

If this code is being touched anyway, might as well partially cover an open issue